interface/new/new_comprehensive_save.php

```diff
@@ -151,8 +147,7 @@
     filter_input(INPUT_POST, 'i2accept_assignment')
 );
 
-$i3dob  = fixDate(filter_input(INPUT_POST, "i3subscriber_DOB"));
-$i3date = fixDate(filter_input(INPUT_POST, "i3effective_date"));
```

Comment:
why were i1effective_date, etc. removed?

Reply:
This field is converted to date format in the 'newInsuranceData()' function (openemr/library/patient.inc L1266)